### PR TITLE
fixed the error returned when uploading a file fails

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-41914

## Description
fixed the error returned when uploading the file failsm to make it more explicit.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/DemistoRESTAPI/ReleaseNotes/1_3_61.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### UploadFile
+
+- Fixed the error returned when the script fails, to be more informative.
```
Comment (ShirleyDenkberg) [inline Packs/DemistoRESTAPI/ReleaseNotes/1_3_61.md:6]:
```suggestion
Fixed the error returned when the script fails, to be more informative.
```

File: Packs/DemistoRESTAPI/Scripts/UploadFile/README.md [hunk 2]
```diff
@@ -21,13 +21,13 @@ This script uses the following commands and scripts.
...
+| incidentID        | Incident ID to upload the file to                                                                                                                                                                                                                  |
+| body              | Request body                                                                                                                                                                                                                                       |
+| target            | Where to upload the file<br/>- Available options are:<br/>- \`war room entry\`: the file will be uploaded as war room entry.<br/>- \`incident attachment\`: the file will be uploaded as incident attachment.<br/>- default are \`war room entry\` |
+| using             | integration instance to use to run the command                                                                                                                                                                                                     |
```
Comment (ShirleyDenkberg) [inline Packs/DemistoRESTAPI/Scripts/UploadFile/README.md:30]:
```suggestion
| using             | Integration instance to use to run the command.                                                                                                                                                                                                     |
```

File: Packs/DemistoRESTAPI/Scripts/UploadFile/README.md [hunk 3]
```diff
@@ -21,13 +21,13 @@ This script uses the following commands and scripts.
...
+|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| entryID           | File entry ID                                                                                                                                                                                                                                      |
+| incidentID        | Incident ID to upload the file to                                                                                                                                                                                                                  |
+| body              | Request body                                                                                                                                                                                                                                       |
```
Comment (ShirleyDenkberg) [inline Packs/DemistoRESTAPI/Scripts/UploadFile/README.md:28]:
```suggestion
| entryID           | File entry ID.                                                                                                                                                                                                                                      |
| incidentID        | Incident ID to upload the file to.                                                                                                                                                                                                                  |
| body              | Request body.                                                                                                                                                                                                                                       |
```

File: Packs/DemistoRESTAPI/Scripts/UploadFile/README.md [hunk 4]
```diff
@@ -21,13 +21,13 @@ This script uses the following commands and scripts.
...
+| target            | Where to upload the file<br/>- Available options are:<br/>- \`war room entry\`: the file will be uploaded as war room entry.<br/>- \`incident attachment\`: the file will be uploaded as incident attachment.<br/>- default are \`war room entry\` |
```
Comment (ShirleyDenkberg) [inline Packs/DemistoRESTAPI/Scripts/UploadFile/README.md:29]:
```suggestion
| target            | Where to upload the file<br/>- Available options are:<br/>- \`war room entry\`: the file will be uploaded as War Room entry.<br/>- \`incident attachment\`: the file will be uploaded as incident attachment.<br/>- default are \`war room entry\` |
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 Doc review completed.

**Review by thefrieddan1 — APPROVED** [review]

**Pull request merged**